src/lib/Behaviors/FillHandleBehavior.tsx

```diff
@@ -75,6 +75,7 @@ export class FillHandleBehavior extends Behavior {
         const selectedCell = selectedCells[i % selectedCells.length];
         return {
           ...cell,
+          checked: selectedCell?.checked ?? false,
```

Comment:
...but I investigated the issue and prepared a patch that solves this issue without modifying the base Cell interface. 
```suggestion
          // ! Patch for checkbox cell template
          // TODO: look for better solution
          ...(cell.type === "checkbox" && { checked: "checked" in selectedCell ? selectedCell.checked : !!selectedCell.value })
```